convex/game/agents.ts

```diff
@@ -106,13 +117,24 @@ export function tickAgent(game: AiTown, now: number, agent: Doc<'agents'>) {
 
   const member = game.conversationMembers.find((m) => m.playerId === player._id);
 
+  const recentlyAttempted =
+    agent.lastInviteAttempt && agent.lastInviteAttempt + ACTIVITY_COOLDOWN > now;
+  const doingActivity = player.activity && player.activity.until > now;
+  // If we're not in a conversation, do something.
+  // If we aren't doing an activity or moving, do something.
+  // If we have been wandering but haven't thought about something to do for
+  // a while, do something.
+  if (!member && !doingActivity && (!player.pathfinding || !recentlyAttempted)) {
+    agent.lastInviteAttempt = now;
```

Comment:
should we rename this to `lastActionAttempt` or something like that?

Reply:
Actually I'm going to change it to only track invite attempts, and not cool down after activities